examples/cd_distillation_cifar10.ipynb

```diff
@@ -0,0 +1,781 @@
+    "\n",
+    "Original CIFAR-10 data:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "(X_train, y_train), (X_test, y_test) = tf.keras.datasets.cifar10.load_data()\n",
+    "X_train = X_train.astype('float32') / 255\n",
+    "X_test = X_test.astype('float32') / 255\n",
+    "y_train = y_train.astype('int64').reshape(-1,)\n",
+    "y_test = y_test.astype('int64').reshape(-1,)\n",
+    "X_train = scale_by_instance(X_train)\n",
+    "X_test = scale_by_instance(X_test)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "For CIFAR-10-C, we can select from the following corruption types at 5 severity levels:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "['brightness', 'contrast', 'defocus_blur', 'elastic_transform', 'fog', 'frost', 'gaussian_blur', 'gaussian_noise', 'glass_blur', 'impulse_noise', 'jpeg_compression', 'motion_blur', 'pixelate', 'saturate', 'shot_noise', 'snow', 'spatter', 'speckle_noise', 'zoom_blur']\n"
+     ]
+    }
+   ],
+   "source": [
+    "corruptions = corruption_types_cifar10c()\n",
+    "print(corruptions)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Let's pick a subset of the corruptions at corruption level 5. Each corruption type consists of perturbations on all of the original test set images."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 8,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "corruption = ['gaussian_noise', 'motion_blur', 'brightness', 'pixelate']\n",
+    "X_corr, y_corr = fetch_cifar10c(corruption=corruption, severity=5, return_X_y=True)\n",
+    "X_corr = X_corr.astype('float32') / 255"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We split the original test set in a reference dataset and a dataset which should not be rejected under the *H<sub>0</sub>* of the K-S test. We also split the corrupted data by corruption type:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 9,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "X_c = []\n",
+    "n_corr = len(corruption)\n",
+    "n_test = X_test.shape[0]\n",
+    "for i in range(n_corr):\n",
+    "    X_c.append(X_corr[i * n_test:(i + 1) * n_test])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We can visualise the same instance for each corruption type:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 10,
+   "metadata": {
+    "tags": [
+     "hide_input"
+    ]
+   },
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "Clipping input data to the valid range for imshow with RGB data ([0..1] for floats or [0..255] for integers).\n"
```

Comment:
*X_test* is scaled by instance, the corrupted datasets in a *[0,1]* range. They should be on the same scale and the plotted instances rescaled in [0,1].